src/Core/Utilities/LoggerFactoryExtensions.cs

```diff
@@ -32,6 +34,16 @@ public static class LoggerFactoryExtensions
                     config.WriteTo.AzureDocumentDB(new Uri(globalSettings.DocumentDb.Uri), globalSettings.DocumentDb.Key,
                         timeToLive: TimeSpan.FromDays(7));
                 }
+                else if(globalSettings.Sentry != null && CoreHelpers.SettingHasValue(globalSettings.Sentry.Dsn))
+                {
+                    config.WriteTo.Sentry(globalSettings.Sentry.Dsn)
+                        .Enrich.FromLogContext()
+                        .Enrich.WithProperty("Project", globalSettings.ProjectName)
+                        .Destructure.With<HttpContextDestructingPolicy>()
+                        .Filter.ByExcluding(e => e.Exception?.CheckIfCaptured() == true);
+
+                    appBuilder.AddSentryContext();
```

Comment:
Looking at the middleware for this : https://github.com/olsh/serilog-sinks-sentry/blob/master/src/Serilog.Sinks.Sentry.AspNetCore/SentrySinkContextMiddleware.cs

It seems like you are going to end up with duplicates of every error logged. We already do the same thing with all exceptions here:  https://github.com/bitwarden/core/blob/master/src/Api/Utilities/ExceptionHandlerFilterAttribute.cs

Reply:
No, there will be no duplicates because you are always setting `context.Result` [here](https://github.com/bitwarden/core/blob/master/src/Api/Utilities/ExceptionHandlerFilterAttribute.cs#L88).
Because the result is always set, the exception will not be re-thrown. So the middleware will never receive the exception.